manifests/r/RevoUninstaller/RevoUninstallerPro/4.3.3/RevoUninstaller.RevoUninstallerPro.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: RevoUninstaller.RevoUninstallerPro
+PackageVersion: 4.3.3
+PackageLocale: en-US
+Publisher: VS Revo Group, Ltd.
+PublisherUrl: https://www.revouninstaller.com
+PublisherSupportUrl: https://www.revouninstaller.com/support
+PrivacyUrl: https://www.revouninstaller.com/revo-uninstaller-privacy-policy
+Author: VS Revo Group, Ltd.
+PackageName: Revo Uninstaller Pro
+PackageUrl: https://www.revouninstaller.com/products/revo-uninstaller-pro
+License: Proprietary
+LicenseUrl: https://www.revouninstaller.com/revo-uninstaller-pro-eula/
+Copyright: Copyright VS Revo Group
+CopyrightUrl: https://www.revouninstaller.com/revo-uninstaller-pro-eula/
+ShortDescription: Revo Uninstaller helps you to uninstall software and remove unwanted programs easily.
+Description: Revo Uninstaller is a freeware uninstall utility. It has powerful features to uninstall programs scanning for leftover files, folders and registry entries after uninstall. With its unique 'Hunter mode' it offers you some simple, easy to use but effective and powerful approaches to manage (uninstall, stop, delete, disable from auto starting) and to get information about your installed and/or running application. Many cleaning tools included!
+Moniker: revo-pro
+Tags:
+- revo
+- revo uninstaller pro
```

```suggestion
- revo-uninstaller-pro
```
from https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#optional-field-10